### PR TITLE
[MDS-5742] CRR Report selection flow

### PR DESCRIPTION
**Pull request by henryoforeh-dev**
## Objective 

[MDS-5742](https://bcmines.atlassian.net/browse/MDS-5742)

- Implemented migration script to add new columns (help_reference_link, cim_or_cpo), update existing columns to accommodate longer character lengths, insert new compliance articles and update some existing compliance articles.

- Implemented functionality to display long description and link to online documentation.

screen1

screen2

### CODE REVIEW

File: services/common/src/components/reports/ReportDetailsForm.tsx [hunk 1]
```diff
@@ -201,6 +204,14 @@ const ReportDetailsForm: FC<ReportDetailsFormProps> = ({
     dispatch(change(FORM.VIEW_EDIT_REPORT, "documents", docs));
   };
 
+  const handleOpenMoreInformation = () => {
+    const newWindow = window.open(
```
Comment (taraepp) [inline services/common/src/components/reports/ReportDetailsForm.tsx]: do we have to use window.open here or can the button be a link instead? I see an example of a Button link in PartyOrgBookForm.tsx (I searched codebase for _blank)

File: migrations/sql/V2024.02.14.12.00__add_help_reference_link.sql [hunk 2]
```diff
@@ -0,0 +1,3151 @@
...
+        'Chief Inspector''s authority to cause work to be done',
+        '2024-02-15',
+        '9999-12-31',
+        'system-mds',
+        current_timestamp,
+        'system-mds',
+        current_timestamp,
+        'Chief Inspector''s authority to cause work to be done',
+        'CIM'
+    );
```
Comment (matbusby-fw) [inline migrations/sql/V2024.02.14.12.00__add_help_reference_link.sql:3151]: 😅 That's a lot of SQL! 🙌🏻

**Review by matbusby-fw — DISMISSED** [review]
Looks great!  Nice work Henry!

File: services/common/src/components/reports/ReportDetailsForm.tsx (hunk 1 above)
Comment (henryoforeh-dev) [reply services/common/src/components/reports/ReportDetailsForm.tsx]: Updated.

**Review by asinn134 — DISMISSED** [review]

**Review by matbusby-fw — APPROVED** [review]

**Review by asinn134 — APPROVED** [review]

**Pull request merged**